tests/arch/arm/arm_interrupt/src/arm_interrupt.c

```diff
@@ -215,15 +215,6 @@ void arm_isr_handler(const void *args)
 		expected_reason = K_ERR_KERNEL_PANIC;
 		__ASSERT(0, "Intentional assert\n");
 	} else if (test_flag == 4) {
-#if defined(CONFIG_CPU_CORTEX_M_HAS_SYSTICK)
-#if !defined(CONFIG_SYS_CLOCK_EXISTS) || !defined(CONFIG_CORTEX_M_SYSTICK)
-		expected_reason = K_ERR_CPU_EXCEPTION;
-		SCB->ICSR |= SCB_ICSR_PENDSTSET_Msk;
-		__DSB();
-		__ISB();
-#endif
-#endif
-	} else if (test_flag == 5) {
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/blob/92e370440e7c97255971678e3af7701ce4e6cc59/tests/arch/arm/arm_interrupt/src/arm_interrupt.c#L203

Noting that `arm_isr_handler` does `test_flag++` prior to checking it, the following will not be properly validated:

https://github.com/zephyrproject-rtos/zephyr/blob/92e370440e7c97255971678e3af7701ce4e6cc59/tests/arch/arm/arm_interrupt/src/arm_interrupt.c#L342

Reply:
Thanks, I had forgotten to remove this. It was added as part of the test-case that I am removing here.
Removed this now.